healpy/src/_sphtools.pyx

```diff
@@ -137,6 +137,11 @@ def map2alm_spin_healpy(maps, spin, lmax = None, mmax = None):
         if mask is not False:
             m[mask] = UNSEEN
 
+    # flip sign for spin 0 to match HEALPix convention
+    if spin == 0:
+        for alm in alms:
+            alm *= -1
+
```

Comment:
can you please explain what was the behaviour before and after this in the docstring? so if a user notice something wrong they can understand what happened.